CMakeLists.txt

```
@@ -90,6 +90,7 @@ option(SSG_PRODUCT_UBUNTU1404 "If enabled, the Ubuntu 14.04 SCAP content will be
 option(SSG_PRODUCT_UBUNTU1604 "If enabled, the Ubuntu 16.04 SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
 option(SSG_PRODUCT_UBUNTU1804 "If enabled, the Ubuntu 18.04 SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
 option(SSG_PRODUCT_VSEL "If enabled, the McAfee VSEL SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
+option(SSG_PRODUCT_WEBMIN "If enabled, the Webmin SCAP content will be built" TRUE)
```

```suggestion
option(SSG_PRODUCT_WEBMIN "If enabled, the Webmin SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
```

Better align with `build_product.sh` script

---

Got it